python/ray/tune/README.rst

```diff
@@ -3,4 +3,26 @@ Ray.tune: Efficient distributed hyperparameter search
 
 Ray.tune is a hyperparameter tuning tool for long-running tasks such as RL and deep learning training.
 
-Documentation can be `found here <https://github.com/ray-project/ray/blob/master/doc/source/tune.rst>`__.
+User documentation can be `found here <https://github.com/ray-project/ray/blob/master/doc/source/tune.rst>`__.
```

Comment:
wouldn't you want to point them towards the `readthedocs`?

Reply:
Fixed